libsrc/telestrat/toascii.s

```diff
@@ -0,0 +1,14 @@
+;
+; char __fastcall__ toascii (char c);
```

Comment:
Please remove this file again.

Background: I wasn't aware until now that ctype.h contans a toascii() declaration. However most target libraries don't have a (null-)implementation so far. This is actually good because the current build system allows to have a common (null-)implementation which is automatically overridden by target-specific (non-null-)implementations. I will provide that common (null-)implementation (and remove the already existing (null-)implementations).

Reply:
>  I will provide that common (null-)implementation (and remove the already existing (null-)implementations).

Done: https://github.com/cc65/cc65/commit/bd9b4ef60c9de6c4b2f0bad2a50d624daf066536